Dockerfile

```diff
@@ -11,21 +11,21 @@ RUN BASEHREF=/ui/dashboard/ PREFIX_API_BASE_URL=/ make build-prod
 WORKDIR /home/node/ui/editor
 RUN BASEHREF=/ui/editor/ make build-prod
 
-FROM golang:1.12.7 
+FROM golang:1.12
 
 COPY .  /go/src/github.com/ovh/utask
 WORKDIR /go/src/github.com/ovh/utask
-RUN make re
-RUN mv hack/Makefile-child Makefile
-
-RUN mkdir -p /app/plugins /app/templates /app/config /app/init /app/static/dashboard /app/static/editor
+RUN make re && \
+    mv hack/Makefile-child Makefile && \
+    mkdir -p /app/plugins /app/templates /app/config /app/init /app/static/dashboard /app/static/editor
 WORKDIR /app
 
 COPY --from=js-builder /home/node/ui/dashboard/dist/utask-ui/*  /app/static/dashboard/
 COPY --from=js-builder /home/node/ui/editor/dist/utask-editor/* /app/static/editor/
+ADD https://raw.githubusercontent.com/vishnubob/wait-for-it/master/wait-for-it.sh /wait-for-it.sh
```

Comment:
Would be better to vendor this script.
Who knows what will be the content of this shell script later ;)

Reply:
Done, thanks!